src/pages/Pools/index.tsx

```diff
@@ -31,7 +31,7 @@ export const Pools = (props: PageProps) => {
   const { title } = page;
   const { network }: any = useApi();
   const navigate = useNavigate();
-  const { bondedPools, membership } = usePools();
+  const { bondedPools, isPooling } = usePools();
```

Comment:
`isPooling` being the equivalent of `isBonding`?

Reply:
Yeah! I'll rename to make it more clear.